packages/react/src/context/use-context-mutator.ts

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useContext, useRef } from 'react';
+import type { EvaluationContext } from '@openfeature/web-sdk';
+import { OpenFeature } from '@openfeature/web-sdk';
+import { Context } from '../common';
+
+export type ContextMutationOptions = {
+  /**
+   * Apply changes to the default context instead of the domain scoped context applied at the <OpenFeatureProvider/>.
+   * Note, if the <OpenFeatureProvider/> has no domain specified, the default is used.
+   */
+  default?: boolean;
```

Comment:
changed from `setGlobal` to `default`.

Reply:
Can you come up with a more descriptive name than `default`?

Reply:
I've changed to `defaultContext`.